src/Controls/StateControl.js

```diff
@@ -324,24 +324,33 @@ class StateControl extends THREE.EventDispatcher {
         viewCoords.copy(this._view.eventToViewCoords(event));
 
         switch (event.pointerType) {
-            case 'mouse':
+            case 'mouse': {
                 this._currentMousePressed = event.button;
-
+                let clickKeyPressed = this._currentKeyPressed;
```

Comment:
I think it would be better to directly assign `this._currentKeyPressed` (for the 3 cases below), given that the corresponding `KeyboardEvent` will be raised later on `onKeyUp` (even when going out of focus).